lib/aux_js.cpp

```diff
@@ -62,13 +62,19 @@ bool startVisualization(const std::string input, const std::string data_type,
    GetAppWindow()->getRenderer().setAntialiasing(0);
 
    std::string line;
+   double minv = 0.0, maxv = 0.0;
    while (ss >> line)
    {
       if (line == "keys")
       {
          std::cout << "parsing 'keys'" << std::endl;
          ss >> stream_state.keys;
       }
+      else if (line == "valuerange")
+      {
+         std::cout << "parsing 'valuerange'" << std::endl;
+         ss >> minv >> maxv;
+      }
```

Comment:
We should (but not now) think about making a shared parser with *glvis.cpp* for the end-of-stream messages

Reply:
I agree, I thought about this too, but didn't have the time to do it carefully.

(plus "valuerange" was the only thing I needed for the additional examples)